### DISCUSSION TITLE
Typescript doesn't work

### DISCUSSION THREAD

**Discussion by aurindo10** [zed-industries/zed#7435] [Feature Requests]
I tried go and it worked fine. But with Typescript I can't get anything. it's like I'm typing code on google doc

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
